standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java

```diff
@@ -1230,7 +1230,7 @@ public void createTable(Table tbl, EnvironmentContext envContext) throws Already
     boolean success = false;
     try {
       // Subclasses can override this step (for example, for temporary tables)
-      create_table_with_environment_context(tbl, envContext);
+       create_table_with_environment_context(tbl, envContext);
```

Comment:
Please do not include formatting only changes.

Reply:
Sure. I will keep in mind. Removed as of now.